docs/dialect.md

```diff
@@ -173,3 +173,87 @@ Allowable date parts: year, quarter, month, dayofyear, day, weekday, week, hou
 
   **Join Capabilities**  <br/>
   Join usage is also defined by capabilities in the manifest file. See the Query section for join relation capabilities [here]({{ site.baseurl }}/docs/capabilities).
+
+### Boolean Support:
+  We can use boolean support functions when a database lacks native boolean support. <br/>
+   **format-true** <br/> 
+  String literal or expression for boolean false value.  
+  ```xml 
+   <format-true value='(1=1)' />
+  ```
+  **format-false** <br/>
+  String literal or expression for boolean false value. 
+  ```xml 
+  <format-false value='(1=0)' />
+  ```
+   **format-bool-as-value** <br/>
+    Used in CASE statements. Determines whether the true or false case is used first. 
+    The value for this property can be: 
+  - __TrueFirst__
+      This is the default case. Logic: `CASE WHEN %1 THEN 1 WHEN NOT %1 THEN 0 ELSE NULL END`
+  - __FalseFirst__
+      False case is used first. Logic: `CASE WHEN NOT %1 THEN 0 WHEN %1 THEN 1 ELSE NULL END`
+
+  ```xml 
+    <format-bool-as-value value='TrueFirst' />
+  ```
+  
+ **Boolean Capabilities**  <br/>
+  
+  Boolean usage is also defined by capabilities in the manifest file. The capabilities are: <br>
+  - __CAP_QUERY_BOOLEXPR_TO_INTEXPR__
+  -  __CAP_QUERY_BOOLEXPR_TO_INTEXPR__
+  - __CAP_QUERY_BOOL_IDENTIFIER_TO_LOGICAL__
+  - __CAP_QUERY_GROUP_BY_BOOL__
+  - __CAP_ODBC_BIND_BOOL_AS_WCHAR_TFLITERAL__ <br/>
+  This capability seems to be necessary only for Hive/Impala connector. 
+- __CAP_ODBC_BIND_BOOL_AS_WCHAR_01LITERAL__ <br/> 
```

Comment:
I think these actually confuse the issue.  Let's stick to the query side stuff for this doc.

Reply:
Removed this part and kept only the link to the details for capabilities